projects/adventure/adv.py

```diff
@@ -28,7 +54,14 @@
 # Fill this out with directions to walk
 # traversal_path = ['n', 'n']
 traversal_path = []
+path = []
+rooms_visited = {}
+reverse_movement = {'n': 's', 's': 'n', 'e': 'w', 'w': 'e'}
 
+rooms_visited[player.current_room] = player.current_room.get_exits()
+
+while len(rooms_visited) < len(room_graph):
```

Comment:
You need to add to the rooms_visited dictionary or else this code will get stuck in an infinite loop.